src/main/kotlin/racingcar/RacingCarManager.kt

```diff
@@ -12,9 +12,7 @@ class RacingCarManager(
     fun startGame() {
         val numberOfCar = inputManager.inputNumberOfCar()
         val numberOfCount = inputManager.inputNumberOfCount()
-        val cars: MutableList<Car> = mutableListOf()
-
-        setCarList(numberOfCar, cars)
+        val cars: List<Car> = createCars(numberOfCar)
```

Comment:
```suggestion
        val cars = createCars(numberOfCar)
```

불변으로 변경해주셨군요! 단, 타입 추론에 대해서는 과감하게 써주셔도 좋을 것 같습니다. 저는 타입 추론은 변수 명과 메서드 명을 더 잘 지어주게 하는 요소라고 생각합니다 :)